src/utilities/route-utils.test.ts

```diff
@@ -210,6 +210,78 @@ describe("RouteUtils", () => {
 
     //#endregion isAbsoluteUrl
 
+    // -----------------------------------------------------------------------------------------
+    // #region queryStringToObject
+    // -----------------------------------------------------------------------------------------
+
+    describe("queryStringToObject", () => {
+        test.each`
+            inputString
+            ${null}
+            ${undefined}
+            ${""}
+            ${" "}
+        `(
+            "given queryString is $inputString, returns empty object",
+            ({ inputString }) => {
+                // Arrange & Act
+                const result = RouteUtils.queryStringToObject<any>(inputString);
+
+                // Assert
+                expect(result).not.toBeNull();
+                expect(typeof result).toBe("object");
```

Comment:
Should we assert that it equals `{}`?

Reply:
Yeah, that would be easy way and get more bang for the buck